applications/mne_fiff_anonymizer/main.cpp

```diff
@@ -0,0 +1,89 @@
+* POSSIBILITY OF SUCH DAMAGE.
+*
+*
+* @brief     Application for anonymizing patient and personal health information from a fiff file..
+*
+*/
+
+
+//*************************************************************************************************************
+//=============================================================================================================
+// INCLUDES
+//=============================================================================================================
+
+#include "fiffanonymizerapp.h"
+#include "commandlineuser.h"
+
+//*************************************************************************************************************
+//=============================================================================================================
+// Eigen
+//=============================================================================================================
+
+
+//*************************************************************************************************************
+//=============================================================================================================
+// QT INCLUDES
+//=============================================================================================================
+
+#include <QCoreApplication>
+#include <QCommandLineParser>
+
+
+//*************************************************************************************************************
+//=============================================================================================================
+// USED NAMESPACES
+//=============================================================================================================
+
+
+//*************************************************************************************************************
+//=============================================================================================================
+// MAIN
+//=============================================================================================================
+
+//=============================================================================================================
+/**
+* The function main marks the entry point of the program.
+* By default, main has the storage class extern.
+*
+* @param [in] argc (argument count) is an integer that indicates how many arguments were entered on the command line when the program was started.
+* @param [in] argv (argument vector) is an array of pointers to arrays of character objects. The array objects are null-terminated strings, representing the arguments that were entered on the command line when the program was started.
+* @return the value that was set to exit() (which is 0 if exit() is called via quit()).
+*/
+int main(int argc, char *argv[])
+{
+    QCoreApplication qtApp(argc, argv);
+    MNEFIFFANONYMIZER::FiffAnonymizerApp app;
+    MNEFIFFANONYMIZER::CommandLineUser user{&app,&qtApp};
```

Comment:
`{...}` style inits are not allowed in C++98. We are dependent on C++98 standard in the minimalVersion since we need to be able to compile the minimalVersion on the old Neuromag acq systems.